packages/graphql/src/schema/get-nodes.ts

```diff
@@ -117,6 +117,7 @@ function getNodes(definitionNodes: DefinitionNodes): Nodes {
             objects: definitionNodes.objectTypes,
             scalars: definitionNodes.scalarTypes,
             unions: definitionNodes.unionTypes,
+            callbacks: options.callbacks,
```

Comment:
**question:** This feels a bit... Weird. You have the function `getObjFieldMeta` which previously accepted purely GraphQL types, and now we're passing in all of these callbacks. Feels like we're slightly polluting the interface, but unsure of a better way around this.